### PR TITLE
Dev 1783 typo

### PR DESCRIPTION
**Pull request by agroblervf**
There was a typo in the Sentiment Analysis Ticket Update
`Customer overal sentiment: ` - should be `overall`

### CODE REVIEW

File: add-ons/contact-lens/packaged.yml [hunk 1]
```diff
@@ -85,7 +85,7 @@ Resources:
         and updates the matching Zendesk support ticket
       Runtime: nodejs12.x
       Handler: index.handler
-      CodeUri: s3://214558022353-connect-zendesk-ap-southeast-2/zendesk-add-ons/contact-lens/b8dc42d486cebb09d843c1462d389e3f
+      CodeUri: s3://214558022353-connect-zendesk-ap-southeast-2/zendesk-add-ons/contact-lens//f06f4120b2a29457f16fa97eeaae66bb
```
Comment (simon-fairall-vf) [inline add-ons/contact-lens/packaged.yml:88]: Is there an extra / in this?

File: add-ons/contact-lens/packaged.yml [hunk 2]
```diff
@@ -48,8 +48,8 @@ Metadata:
       Zendesk Support ticket. To be used with the Amazon Connect app for Zendesk.
     Author: VoiceFoundry-APAC
     SpdxLicenseId: GPL-3.0-or-later
-    LicenseUrl: s3://214558022353-connect-zendesk-ap-southeast-2/zendesk-add-ons/contact-lens/e62637ea8a114355b985fd86c9ffbd6e
-    ReadmeUrl: s3://214558022353-connect-zendesk-ap-southeast-2/zendesk-add-ons/contact-lens/c8fbd6d06b83a4dbae7bb792ec772357
+    LicenseUrl: s3://214558022353-connect-zendesk-ap-southeast-2/zendesk-add-ons/contact-lens//e62637ea8a114355b985fd86c9ffbd6e
```
Comment (simon-fairall-vf) [inline add-ons/contact-lens/packaged.yml:51]: Extra /?

File: add-ons/contact-lens/packaged.yml (hunk 1 above)
Comment (agroblervf) [reply add-ons/contact-lens/packaged.yml]: Good spot, thanks! Also took the opportunity to increment the version number.

**Review by simon-fairall-vf — APPROVED** [review]

**Review by AlexD1979 — APPROVED** [review]
Looks good as the // are solved.

**Pull request merged**